DigitalLearningSolutions.Web/Helpers/CentreCustomPromptHelper.cs

```diff
@@ -1,117 +1,117 @@
-﻿namespace DigitalLearningSolutions.Web.Helpers

```

Comment:
Can you check line ending on this file (they may have been wrong before?)

Reply:
Hmm, says CRLF for me

Reply:
That's fine then. They may have been LF before and you've fixed it.

Reply:
It's probably actually just the file name change

Reply:
This shows as LF for me when I check out the branch

Reply:
Ah I had git auto crlf on, all the LF files should be CRLF now